code-challenges/CC3_sort/challenges-03.test.js

```diff
@@ -49,15 +49,11 @@ For example, ['Alphabet', 'alphabet', 'carrot', 'Zebra'] is correctly sorted, an
 
 const alphabetizeBetter = (arr) => {
   arr.sort((a, b) => {
-    if (a.toUpperCase() > b.toUpperCase()) return 1;
     if (a.toUpperCase() < b.toUpperCase()) return -1;
```

Comment:
You do want the arrow function inside the sort method to always have a return value. Right now you are only returning if a.toUppercase() < b.toUpperCase(). So the lines you previously had, but deleted, should be brought back in.